Scripts/Mac/RunFunctionalTests.sh

```diff
@@ -7,9 +7,8 @@ if [ -z $CONFIGURATION ]; then
   CONFIGURATION=Debug
 fi
 
-sudo mkdir /GVFS.FT
-sudo chown $USER /GVFS.FT
-
+sudo mkdir ~/GVFS.FT
```

Comment:
Does this still need `sudo`?

Reply:
Incidentally, the `-p` argument to `mkdir` might be appropriate here to avoid spamming an error when the directory already exists.

Reply:
@pmj I was also thinking "-p" might reduce noise.  This appears to be existing.  @wilbaker  any reason why this didn't have the "-p" arg previously?

Reply:
> any reason why this didn't have the "-p" arg previously?

I suspect it was that we either didn't know about the option or didn't mind the noise.  

`-p` will also create intermediate directories if they're missing and so the absence of `-p` could be a sanity check that the intermediate directories that we expect to be present are in fact present.

Reply:
`-p` will also create intermediate directories if they're missing and so the absence of -p could be a sanity check that the intermediate directories that we expect to be present are in fact present.

I could see that.  Since this is pre-existing and outside the scope of my changes I'll leave it be.